### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Vishal30M**
In this branch I did:
Edit index.html 
Add nav menu
Edit style.css
Give a responsive style to the website
Check for linter errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,9 +65,9 @@ <h3 class="contact">LET'S CONNECT </h3>
...
     <section class="Projects">
         <div class="card">
-
+            <img class="snapshot" src="images/SS0.png" alt="snapshot">
             <div class="right">
-                <img class="snapshot" src="images/S0.png" alt="snapshot">
+
                 <h2>Tonic</h2>
                 <ul class="detail">
                     <li class="main">CANOPY</li>
```
Comment (nahnahdev) [inline index.html]: - Great job so far @Vishal30M There are just a few changes we need to make and your project will be complete :+1: 
- [ ] Each of your cards is missing some styles as seen in the Figma design. As a result, they don't look exactly like the Figma design. I have pointed them out in the image below so kindly fix these :wink: 
- Your Design
image

- The Figma Design
image

File: index.html [hunk 2]
```diff
@@ -159,105 +166,108 @@ <h2>Multi-Post Stories</h2>
...
+                                <i class="fa fa-linkedin fa-2x" aria-hidden="true"></i>
+                            </a>
+                        </li>
+                        <li><a href="#">
+                                <i class="fa fa-medium fa-2x" aria-hidden="true"></i>
+                            </a>
+                        </li>
+                        <li><a href="#">
+                                <i class="fa fa-github fa-2x" aria-hidden="true"></i>
+                            </a>
```
Comment (nahnahdev) [inline index.html]: - [ ] Please ensure that the `about me` part takes 40% of the page and the `languages, skills and frameworks'` part takes the other 60% of the section as shown in the Figma Design :100: 

- [ ] There are also some other elements that need to be appropriately styled so your design looks similar to the Figma design. I have pointed them out in the image below :100: 

- Your Design
image

- The Figma Design
image

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @Vishal30M , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Nice Design :heavy_check_mark:
- The mobile design looks good :heavy_check_mark:
- Professional README :heavy_check_mark:
- No linter errors :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Giving your navbar a fixed position in both desktop and mobile :100: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @Vishal30M, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**